ingestion/functions/parsing/brazil_espirito_santo/espirito_santo.py

```diff
@@ -232,7 +229,7 @@ def convert_demographics(gender: str, age: str, ethnicity: str, healthcare_proff
             demo["ageRange"] = {"start": float(age_range[0]), "end": float(age_range[2][:2])}
     if ethnicity:
         demo["ethnicity"] = convert_ethnicity(ethnicity)
-    if healthcare_proffesional not in _NONE_TYPES:
+    if healthcare_proffesional == "Sim":
```

Comment:
there's a typo here, `healthcare_professional`. It doesn't need to hold up the PR but would be nice to fix.

Reply:
Cool, will merge for now so that Zoe can fix the ingestion, but will tackle this next week.